charts/nfs-subdir-external-provisioner/README.md

```diff
@@ -69,7 +69,7 @@ The following tables lists the configurable parameters of this chart and their d
 | `leaderElection.enabled`            | Enables or disables leader election                                                                   | `true`                                                   |
 | `nfs.server`                        | Hostname of the NFS server (required)                                                                 | null (ip or hostname)                                    |
 | `nfs.path`                          | Basepath of the mount point to be used                                                                | `/nfs-storage`                                           |
-| `nfs.mountOptions`                  | Mount options (e.g. 'nfsvers=3')                                                                      | null                                                     |
+| `nfs.mountOptions`                  | Mount options (e.g. {"nolock"\,"nfsvers=3"})                                                          | null                                                     |
```

Comment:
when file is rendered, the `\` disappears, as you can see here: https://github.com/kubernetes-sigs/nfs-subdir-external-provisioner/blob/27d772798deb510c105b05618686f35cf0cb71ec/charts/nfs-subdir-external-provisioner/README.md

please wrap it with \` like: `{"nolock"\,"nfsvers=3"}`

```
`{"nolock"\,"nfsvers=3"}`
```